### DISCUSSION TITLE
v7.12.2

### DISCUSSION THREAD

**Discussion by gkatsev** [videojs/video.js#7239] [Releases]
<a name="7.12.2"></a>
### Bug Fixes

* update to VHS 2.8.1 ([#7238](https://github.com/videojs/video.js/issues/7238)) ([c4cfa55](https://github.com/videojs/video.js/commit/c4cfa55))
* **utils:** add try and catch for computedStyle ([#7214](https://github.com/videojs/video.js/issues/7214)) ([90ce2d7](https://github.com/videojs/video.js/commit/90ce2d7))
* Better text for exit fullscreen ([#7183](https://github.com/videojs/video.js/issues/7183)) ([0e46624](https://github.com/videojs/video.js/commit/0e46624))
* Don't hide menus with one item and a title ([#7215](https://github.com/videojs/video.js/issues/7215)) ([d4a08de](https://github.com/videojs/video.js/commit/d4a08de))
* exit full window mode with Esc key ([#7224](https://github.com/videojs/video.js/issues/7224)) ([e9953e5](https://github.com/videojs/video.js/commit/e9953e5))
* incorrect focus styles on selected MenuItem ([#7202](https://github.com/videojs/video.js/issues/7202)) ([06cdb6f](https://github.com/videojs/video.js/commit/06cdb6f)), closes [#7200](https://github.com/videojs/video.js/issues/7200)
* make Playback Rate control work better with screen readers ([#7193](https://github.com/videojs/video.js/issues/7193)) ([17919ce](https://github.com/videojs/video.js/commit/17919ce)), closes [#7121](https://github.com/videojs/video.js/issues/7121)
* silence play promise in the play toggle. ([#7189](https://github.com/videojs/video.js/issues/7189)) ([2c6e439](https://github.com/videojs/video.js/commit/2c6e439)), closes [#6998](https://github.com/videojs/video.js/issues/6998)
* user and programmatic seeks with live streams ([#7210](https://github.com/videojs/video.js/issues/7210)) ([39485fc](https://github.com/videojs/video.js/commit/39485fc))

### Chores

* update 'global' package in dependencies ([#7213](https://github.com/videojs/video.js/issues/7213)) ([cb1d29b](https://github.com/videojs/video.js/commit/cb1d29b))
* update node/nvmrc and various dependencies ([#7221](https://github.com/videojs/video.js/issues/7221)) ([90f3e39](https://github.com/videojs/video.js/commit/90f3e39)), closes [#7216](https://github.com/videojs/video.js/issues/7216) [#6933](https://github.com/videojs/video.js/issues/6933) [#6924](https://github.com/videojs/video.js/issues/6924) [#7179](https://github.com/videojs/video.js/issues/7179)

<hr /><em>This discussion was created from the release <a href='https://github.com/videojs/video.js/releases/tag/v7.12.2'>v7.12.2</a>.</em>

**Comment by phloxic** [comment]
```
curl -q https://cdnjs.cloudflare.com/ajax/libs/video.js/7.12.2/video.min.js
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
curl -I https://vjs.zencdn.net/7.12.2/video.min.js | head -1
HTTP/2 403
```
;-)

**Reply by gkatsev** [reply]
I'm not sure why cdnjs doesn't have it yet. It is up on unpkg, for example: https://unpkg.com/video.js@7.12.2/dist/video.min.js
As for our CDN, I forgot to publish it there. I'll get around to it some time today.

**Reply by phloxic** [reply]
No worries, just wanted to let you know.

**Reply by gkatsev** [reply]
Published some more stuff to the Video.js CDN. Seems like cdnjs still doesn't have this version, and I'm really not sure why.

**Reply by phloxic** [reply]
FWIW, I can confirm that vjs.zencdn.net now serves 7.12.2 and 7.12.3. Thank you.

**Reply by gkatsev** [reply]
Oh, I wonder if it's because this is a pre-release, technically. Going to promote it to 7.12.3 to latest.
